utils/utils.ts

```diff
@@ -11,7 +11,11 @@ import { getAccountJSON } from "./frontend/utils/Helper";
 
 export const MGA_ASSET_ID = new BN(0);
 export const MGA_ASSET_NAME = "MGA";
+export const ETH_ASSET_ID = new BN(1);
```

Comment:
It could be worth moving these asset names and IDs to an external config file

Reply:
done!
Created Constants.ts file.